src/middlewares/validationHandler/guestValidator.js

```diff
@@ -0,0 +1,25 @@
+import { check, validationResult } from "express-validator";
+
+exports.guestValidator = [
+    check('name').trim().not().isEmpty()
+    .withMessage("Please provide your Name")
+    .isLength({min: 5, max: 50})
+    .withMessage("Name must be within 5 to 50 Characters"),
+    check('email').normalizeEmail().isEmail()
+    .withMessage("Please provide a valid Email"),
+    check('phone').trim().not().isEmpty()
+    .withMessage("Please provide your Phone Number")
+    .isLength({min: 11, max: 11})
+    .withMessage("Please provide a valid Phone Number"),
+    check('gender').not().isEmpty()
+    .withMessage("Please Select your Gender")
+    
+]
+
+exports.guestValidation = (req, res, next) => {
```

Comment:
Move this to a helper file, all custom functions from your validators should be in a helper file, for easy readability, and reference

Reply:
I have moved it.